pycm/curve.py

```diff
@@ -183,6 +188,23 @@ def __repr__(self) -> str:
         """Representation method."""
         return "pycm.ROCCurve(classes: " + str(self.classes) + ")"
 
+    def optimal_thresholds(self) -> Dict[Any, float]:
+        """
+        Get optimal thresholds for each class.
```

Comment:
This is more like a philosophical question: is there a way we can find a threshold that optimizes for all classes at the same time (e.g., like $\arg\min_x \sum_{c \in class} √[(1-TPR_c(x))² + (FPR_c(x))²]$) if you think there's such a psobility I suggest changing this paramter to `class_optimal_thresholds` so we have this `optimal_thresholds` (or `optimal_threshold`) saved for future.

Reply:
In this particular instance regarding the ROC Curve, this possibility is quite rare.